### ISSUE TITLE
Error while creating service

### ISSUE THREAD

**Issue by dozieogbo** [dokku/dokku-elasticsearch#91]
## Description of problem
Whenever I run `dokku elasticsearch:create api`, I get the error below.

```
=====> Extracting config files
stat -: permission denied
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
```

## Environment Information
docker version:
       Client: Docker Engine - Community
        Version:           19.03.13
        API version:       1.40
        Go version:        go1.13.15
        Git commit:        4484c46d9d
        Built:             Wed Sep 16 17:02:52 2020
        OS/Arch:           linux/amd64
        Experimental:      false

dokku version: dokku version 0.21.4

dokku plugins:
       elasticsearch        1.15.1 enabled    dokku elasticsearch service plugin

**Comment by josegonzalez** [comment]
Can you try it with:

```
dokku --trace elasticsearch:create api
```

And gist the output?

**Comment by dozieogbo** [comment]
Does this help?

```
+ export DOKKU_HOST_ROOT=/home/dokku
+ DOKKU_HOST_ROOT=/home/dokku
+ export DOKKU_DISTRO
++ . /etc/os-release
++ echo ubuntu
+ DOKKU_DISTRO=ubuntu
+ export DOCKER_BIN=docker
+ DOCKER_BIN=docker
+ export DOKKU_IMAGE=gliderlabs/herokuish:latest
+ DOKKU_IMAGE=gliderlabs/herokuish:latest
+ export DOKKU_LIB_ROOT=/var/lib/dokku
+ DOKKU_LIB_ROOT=/var/lib/dokku
+ export PLUGIN_PATH=/var/lib/dokku/plugins
+ PLUGIN_PATH=/var/lib/dokku/plugins
+ export PLUGIN_AVAILABLE_PATH=/var/lib/dokku/plugins/available
+ PLUGIN_AVAILABLE_PATH=/var/lib/dokku/plugins/available
+ export PLUGIN_ENABLED_PATH=/var/lib/dokku/plugins/enabled
+ PLUGIN_ENABLED_PATH=/var/lib/dokku/plugins/enabled
+ export PLUGIN_CORE_PATH=/var/lib/dokku/core-plugins
+ PLUGIN_CORE_PATH=/var/lib/dokku/core-plugins
+ export PLUGIN_CORE_AVAILABLE_PATH=/var/lib/dokku/core-plugins/available
+ PLUGIN_CORE_AVAILABLE_PATH=/var/lib/dokku/core-plugins/available
+ export PLUGIN_CORE_ENABLED_PATH=/var/lib/dokku/core-plugins/enabled
+ PLUGIN_CORE_ENABLED_PATH=/var/lib/dokku/core-plugins/enabled
+ export DOKKU_SYSTEM_GROUP=dokku
+ DOKKU_SYSTEM_GROUP=dokku
+ export DOKKU_SYSTEM_USER=dokku
+ DOKKU_SYSTEM_USER=dokku
+ export DOKKU_API_VERSION=1
+ DOKKU_API_VERSION=1
+ export DOKKU_NOT_IMPLEMENTED_EXIT=10
+ DOKKU_NOT_IMPLEMENTED_EXIT=10
+ export DOKKU_VALID_EXIT=0
+ DOKKU_VALID_EXIT=0
+ export DOKKU_PID=280797
+ DOKKU_PID=280797
+ export DOKKU_LOGS_DIR=/var/log/dokku
+ DOKKU_LOGS_DIR=/var/log/dokku
+ export DOKKU_EVENTS_LOGFILE=/var/log/dokku/events.log
+ DOKKU_EVENTS_LOGFILE=/var/log/dokku/events.log
+ export DOKKU_CONTAINER_LABEL=dokku
+ DOKKU_CONTAINER_LABEL=dokku
+ export 'DOKKU_GLOBAL_BUILD_ARGS=--label=org.label-schema.schema-version=1.0 --label=org.label-schema.vendor=dokku --label=dokku'
+ DOKKU_GLOBAL_BUILD_ARGS='--label=org.label-schema.schema-version=1.0 --label=org.label-schema.vendor=dokku --label=dokku'
+ export 'DOKKU_GLOBAL_RUN_ARGS=--label=org.label-schema.schema-version=1.0 --label=org.label-schema.vendor=dokku --label=dokku'
+ DOKKU_GLOBAL_RUN_ARGS='--label=org.label-schema.schema-version=1.0 --label=org.label-schema.vendor=dokku --label=dokku'
+ source /var/lib/dokku/core-plugins/available/common/functions
++ set -eo pipefail
++ [[ -n 1 ]]
++ set -x
+ parse_args elasticsearch:create api
+ declare 'desc=top-level cli arg parser'
+ local next_index=1
+ local skip=false
+ args=("$@")
+ local args
+ local flags
+ for arg in "$@"
+ [[ false == \t\r\u\e ]]
+ case "$arg" in
+ [[ false == \t\r\u\e ]]
+ [[ elasticsearch:create == \-\-\a\p\p ]]
+ [[ elasticsearch:create =~ ^--.* ]]
+ next_index=2
+ for arg in "$@"
+ [[ false == \t\r\u\e ]]
+ case "$arg" in
+ [[ false == \t\r\u\e ]]
+ [[ api == \-\-\a\p\p ]]
+ [[ api =~ ^--.* ]]
+ next_index=3
+ [[ -z --trace ]]
+ return 0
+ args=("$@")
+ skip_arg=false
+ [[ elasticsearch:create =~ ^--.* ]]
+ has_tty
+ declare 'desc=return 0 if we have a tty'
++ LC_ALL=C
++ /usr/bin/tty
+ [[ /dev/pts/0 == \n\o\t\ \a\ \t\t\y ]]
+ return 0
++ id -un
+ [[ dokku != \d\o\k\k\u ]]
+ [[ elasticsearch:create =~ ^plugin:.* ]]
+ [[ elasticsearch:create == \s\s\h\-\k\e\y\s\:\a\d\d ]]
+ [[ elasticsearch:create == \s\s\h\-\k\e\y\s\:\r\e\m\o\v\e ]]
+ [[ -n '' ]]
+ dokku_auth elasticsearch:create api
+ declare 'desc=calls user-auth plugin trigger'
+ export SSH_USER=root
+ SSH_USER=root
+ export SSH_NAME=default
+ SSH_NAME=default
++ wc -l
++ find /var/lib/dokku/plugins/enabled/20_events/user-auth
+ [[ 1 == 1 ]]
+ return 0
+ case "$1" in
+ execute_dokku_cmd elasticsearch:create api
+ declare 'desc=executes dokku sub-commands'
+ local PLUGIN_NAME=elasticsearch:create
+ local PLUGIN_CMD=elasticsearch:create
+ local implemented=0
+ local script
+ argv=("$@")
+ local argv
+ case "$PLUGIN_NAME" in
++ readlink -f /var/lib/dokku/plugins/enabled/elasticsearch
+ [[ /var/lib/dokku/plugins/available/elasticsearch == *core-plugins* ]]
+ [[ -x /var/lib/dokku/plugins/enabled/elasticsearch:create/subcommands/default ]]
+ [[ -x /var/lib/dokku/plugins/enabled/elasticsearch:create/subcommands/elasticsearch:create ]]
+ [[ -x /var/lib/dokku/plugins/enabled/elasticsearch/subcommands/create ]]
+ [[ -n create ]]
+ /var/lib/dokku/plugins/enabled/elasticsearch/subcommands/create elasticsearch:create api
+ source /var/lib/dokku/plugins/enabled/common/functions
++ set -eo pipefail
++ [[ -n 1 ]]
++ set -x
++++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/subcommands/create
+++ cd /var/lib/dokku/plugins/enabled/elasticsearch/subcommands
+++ pwd
++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/subcommands
+ source /var/lib/dokku/plugins/enabled/elasticsearch/functions
++++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/functions
+++ cd /var/lib/dokku/plugins/enabled/elasticsearch
+++ pwd
++ source /var/lib/dokku/plugins/enabled/elasticsearch/config
+++++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/config
++++ cd /var/lib/dokku/plugins/enabled/elasticsearch
++++ pwd
+++ _DIR=/var/lib/dokku/plugins/enabled/elasticsearch
+++ export ELASTICSEARCH_IMAGE=elasticsearch
+++ ELASTICSEARCH_IMAGE=elasticsearch
+++ export ELASTICSEARCH_IMAGE_VERSION=7.14.1
+++ ELASTICSEARCH_IMAGE_VERSION=7.14.1
+++ export ELASTICSEARCH_ROOT=/var/lib/dokku/services/elasticsearch
+++ ELASTICSEARCH_ROOT=/var/lib/dokku/services/elasticsearch
+++ export ELASTICSEARCH_HOST_ROOT=/var/lib/dokku/services/elasticsearch
+++ ELASTICSEARCH_HOST_ROOT=/var/lib/dokku/services/elasticsearch
+++ PLUGIN_UNIMPLEMENTED_SUBCOMMANDS=("backup" "backup-auth" "backup-deauth" "backup-schedule" "backup-schedule-cat" "backup-set-encryption" "backup-unschedule" "backup-unset-encryption" "clone" "connect" "export" "import")
+++ export PLUGIN_UNIMPLEMENTED_SUBCOMMANDS
+++ export PLUGIN_COMMAND_PREFIX=elasticsearch
+++ PLUGIN_COMMAND_PREFIX=elasticsearch
+++ export PLUGIN_CONFIG_ROOT=/var/lib/dokku/config/elasticsearch
+++ PLUGIN_CONFIG_ROOT=/var/lib/dokku/config/elasticsearch
+++ export PLUGIN_DATA_ROOT=/var/lib/dokku/services/elasticsearch
+++ PLUGIN_DATA_ROOT=/var/lib/dokku/services/elasticsearch
+++ export PLUGIN_DATA_HOST_ROOT=/var/lib/dokku/services/elasticsearch
+++ PLUGIN_DATA_HOST_ROOT=/var/lib/dokku/services/elasticsearch
+++ PLUGIN_DATASTORE_PORTS=(9200 9300)
+++ export PLUGIN_DATASTORE_PORTS
+++ export PLUGIN_DATASTORE_WAIT_PORT=9200
+++ PLUGIN_DATASTORE_WAIT_PORT=9200
+++ export PLUGIN_DEFAULT_ALIAS=ELASTICSEARCH
+++ PLUGIN_DEFAULT_ALIAS=ELASTICSEARCH
+++ export PLUGIN_DISABLE_PULL=
+++ PLUGIN_DISABLE_PULL=
+++ export PLUGIN_DISABLE_PULL_VARIABLE=ELASTICSEARCH_DISABLE_PULL
+++ PLUGIN_DISABLE_PULL_VARIABLE=ELASTICSEARCH_DISABLE_PULL
+++ export PLUGIN_ALT_ALIAS=DOKKU_ELASTICSEARCH
+++ PLUGIN_ALT_ALIAS=DOKKU_ELASTICSEARCH
+++ export PLUGIN_IMAGE=elasticsearch
+++ PLUGIN_IMAGE=elasticsearch
+++ export PLUGIN_IMAGE_VERSION=7.14.1
+++ PLUGIN_IMAGE_VERSION=7.14.1
+++ export PLUGIN_SCHEME=http
+++ PLUGIN_SCHEME=http
+++ export PLUGIN_SERVICE=Elasticsearch
+++ PLUGIN_SERVICE=Elasticsearch
+++ export PLUGIN_VARIABLE=ELASTICSEARCH
+++ PLUGIN_VARIABLE=ELASTICSEARCH
+++ export PLUGIN_BASE_PATH=/var/lib/dokku/plugins
+++ PLUGIN_BASE_PATH=/var/lib/dokku/plugins
+++ export PLUGIN_CONFIG_SUFFIX=config
+++ PLUGIN_CONFIG_SUFFIX=config
+++ [[ -n 1 ]]
+++ export PLUGIN_BASE_PATH=/var/lib/dokku/plugins/enabled
+++ PLUGIN_BASE_PATH=/var/lib/dokku/plugins/enabled
+++ export PLUGIN_BUSYBOX_IMAGE=busybox:1.31.1-uclibc
+++ PLUGIN_BUSYBOX_IMAGE=busybox:1.31.1-uclibc
+++ export PLUGIN_AMBASSADOR_IMAGE=dokku/ambassador:0.3.3
+++ PLUGIN_AMBASSADOR_IMAGE=dokku/ambassador:0.3.3
+++ export PLUGIN_S3BACKUP_IMAGE=dokku/s3backup:0.10.3
+++ PLUGIN_S3BACKUP_IMAGE=dokku/s3backup:0.10.3
+++ export PLUGIN_WAIT_IMAGE=dokku/wait:0.4.3
+++ PLUGIN_WAIT_IMAGE=dokku/wait:0.4.3
+++ export ELASTICSEARCH_CONFIG_OPTIONS=
+++ ELASTICSEARCH_CONFIG_OPTIONS=
++ set -eo pipefail
++ [[ -n 1 ]]
++ set -x
++++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/functions
+++ cd /var/lib/dokku/plugins/enabled/elasticsearch
+++ pwd
++ source /var/lib/dokku/plugins/enabled/elasticsearch/common-functions
+++++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/common-functions
++++ cd /var/lib/dokku/plugins/enabled/elasticsearch
++++ pwd
+++ source /var/lib/dokku/plugins/enabled/elasticsearch/config
++++++ dirname /var/lib/dokku/plugins/enabled/elasticsearch/config
+++++ cd /var/lib/dokku/plugins/enabled/elasticsearch
+++++ pwd
++++ _DIR=/var/lib/dokku/plugins/enabled/elasticsearch
++++ export ELASTICSEARCH_IMAGE=elasticsearch
++++ ELASTICSEARCH_IMAGE=elasticsearch
++++ export ELASTICSEARCH_IMAGE_VERSION=7.14.1
++++ ELASTICSEARCH_IMAGE_VERSION=7.14.1
++++ export ELASTICSEARCH_ROOT=/var/lib/dokku/services/elasticsearch
++++ ELASTICSEARCH_ROOT=/var/lib/dokku/services/elasticsearch
++++ export ELASTICSEARCH_HOST_ROOT=/var/lib/dokku/services/elasticsearch
++++ ELASTICSEARCH_HOST_ROOT=/var/lib/dokku/services/elasticsearch
++++ PLUGIN_UNIMPLEMENTED_SUBCOMMANDS=("backup" "backup-auth" "backup-deauth" "backup-schedule" "backup-schedule-cat" "backup-set-encryption" "backup-unschedule" "backup-unset-encryption" "clone" "connect" "export" "import")
++++ export PLUGIN_UNIMPLEMENTED_SUBCOMMANDS
++++ export PLUGIN_COMMAND_PREFIX=elasticsearch
++++ PLUGIN_COMMAND_PREFIX=elasticsearch
++++ export PLUGIN_CONFIG_ROOT=/var/lib/dokku/config/elasticsearch
++++ PLUGIN_CONFIG_ROOT=/var/lib/dokku/config/elasticsearch
++++ export PLUGIN_DATA_ROOT=/var/lib/dokku/services/elasticsearch
++++ PLUGIN_DATA_ROOT=/var/lib/dokku/services/elasticsearch
++++ export PLUGIN_DATA_HOST_ROOT=/var/lib/dokku/services/elasticsearch
++++ PLUGIN_DATA_HOST_ROOT=/var/lib/dokku/services/elasticsearch
++++ PLUGIN_DATASTORE_PORTS=(9200 9300)
++++ export PLUGIN_DATASTORE_PORTS
++++ export PLUGIN_DATASTORE_WAIT_PORT=9200
++++ PLUGIN_DATASTORE_WAIT_PORT=9200
++++ export PLUGIN_DEFAULT_ALIAS=ELASTICSEARCH
++++ PLUGIN_DEFAULT_ALIAS=ELASTICSEARCH
++++ export PLUGIN_DISABLE_PULL=
++++ PLUGIN_DISABLE_PULL=
++++ export PLUGIN_DISABLE_PULL_VARIABLE=ELASTICSEARCH_DISABLE_PULL
++++ PLUGIN_DISABLE_PULL_VARIABLE=ELASTICSEARCH_DISABLE_PULL
++++ export PLUGIN_ALT_ALIAS=DOKKU_ELASTICSEARCH
++++ PLUGIN_ALT_ALIAS=DOKKU_ELASTICSEARCH
++++ export PLUGIN_IMAGE=elasticsearch
++++ PLUGIN_IMAGE=elasticsearch
++++ export PLUGIN_IMAGE_VERSION=7.14.1
++++ PLUGIN_IMAGE_VERSION=7.14.1
++++ export PLUGIN_SCHEME=http
++++ PLUGIN_SCHEME=http
++++ export PLUGIN_SERVICE=Elasticsearch
++++ PLUGIN_SERVICE=Elasticsearch
++++ export PLUGIN_VARIABLE=ELASTICSEARCH
++++ PLUGIN_VARIABLE=ELASTICSEARCH
++++ export PLUGIN_BASE_PATH=/var/lib/dokku/plugins
++++ PLUGIN_BASE_PATH=/var/lib/dokku/plugins
++++ export PLUGIN_CONFIG_SUFFIX=config
++++ PLUGIN_CONFIG_SUFFIX=config
++++ [[ -n 1 ]]
++++ export PLUGIN_BASE_PATH=/var/lib/dokku/plugins/enabled
++++ PLUGIN_BASE_PATH=/var/lib/dokku/plugins/enabled
++++ export PLUGIN_BUSYBOX_IMAGE=busybox:1.31.1-uclibc
++++ PLUGIN_BUSYBOX_IMAGE=busybox:1.31.1-uclibc
++++ export PLUGIN_AMBASSADOR_IMAGE=dokku/ambassador:0.3.3
++++ PLUGIN_AMBASSADOR_IMAGE=dokku/ambassador:0.3.3
++++ export PLUGIN_S3BACKUP_IMAGE=dokku/s3backup:0.10.3
++++ PLUGIN_S3BACKUP_IMAGE=dokku/s3backup:0.10.3
++++ export PLUGIN_WAIT_IMAGE=dokku/wait:0.4.3
++++ PLUGIN_WAIT_IMAGE=dokku/wait:0.4.3
++++ export ELASTICSEARCH_CONFIG_OPTIONS=
++++ ELASTICSEARCH_CONFIG_OPTIONS=
+++ set -eo pipefail
+++ [[ -n 1 ]]
+++ set -x
+++ source /var/lib/dokku/plugins/available/config/functions
++++ set -eo pipefail
++++ [[ -n 1 ]]
++++ set -x
++++ source /var/lib/dokku/core-plugins/available/common/functions
+++++ set -eo pipefail
+++++ [[ -n 1 ]]
+++++ set -x
++ source /var/lib/dokku/plugins/enabled/common/functions
+++ set -eo pipefail
+++ [[ -n 1 ]]
+++ set -x
++ source /var/lib/dokku/plugins/available/config/functions
+++ set -eo pipefail
+++ [[ -n 1 ]]
+++ set -x
+++ source /var/lib/dokku/core-plugins/available/common/functions
++++ set -eo pipefail
++++ [[ -n 1 ]]
++++ set -x
++ [[ -f /var/lib/dokku/plugins/available/docker-options/functions ]]
++ source /var/lib/dokku/plugins/available/docker-options/functions
+++ set -eo pipefail
+++ [[ -n 1 ]]
+++ set -x
+++ source /var/lib/dokku/core-plugins/available/common/functions
++++ set -eo pipefail
++++ [[ -n 1 ]]
++++ set -x
+++ AVAILABLE_PHASES=(build deploy run)
+ service-create-cmd elasticsearch:create api
+ declare 'desc=create a Elasticsearch service'
+ argv=("$@")
+ local cmd=elasticsearch:create argv
+ [[ elasticsearch:create == \e\l\a\s\t\i\c\s\e\a\r\c\h\:\c\r\e\a\t\e ]]
+ shift 1
+ CREATE_FLAGS_LIST=("${@:2}")
+ declare SERVICE=api CREATE_FLAGS_LIST
+ service_create api
+ local SERVICE=api
+ is_valid_service_name api
+ declare 'desc=validate a service name'
+ declare SERVICE=api
+ [[ -z api ]]
+ [[ api =~ ^[A-Za-z0-9_-]+$ ]]
+ return 0
+ [[ -z api ]]
+ [[ ! -d /var/lib/dokku/services/elasticsearch/api ]]
+ SERVICE_ROOT=/var/lib/dokku/services/elasticsearch/api
+ LINKS_FILE=/var/lib/dokku/services/elasticsearch/api/LINKS
+ service_parse_args
+ declare 'desc=cli arg parser'
+ local next_index=1
+ local skip=false
+ args=("$@")
+ local args
+ OPTIND=1
+ getopts a:c:C:d:i:I:m:p:q:R:r:s:u: opt
+ shift 0
++ echo 7.14.1
++ cut -d . -f 1
+ local ELASTICSEARCH_MAJOR_VERSION=7
+ [[ 7 -ne 2 ]]
+ service_check_sysctl
++ sysctl -n vm.max_map_count
+ local VM_MAX_MAP_COUNT_CURRENT=262144
+ local VM_MAX_MAP_COUNT_REQUIRED=262144
+ '[' 262144 -lt 262144 ']'
+ service_image_exists api
+ declare 'desc=check if the current image exists'
+ declare SERVICE=api
+ local plugin_image=elasticsearch
+ local plugin_image_version=7.14.1
+ [[ -f /var/lib/dokku/services/elasticsearch/api/IMAGE ]]
+ [[ -f /var/lib/dokku/services/elasticsearch/api/IMAGE_VERSION ]]
+ local IMAGE=elasticsearch:7.14.1
++ docker images -q elasticsearch:7.14.1
+ [[ f287f2cfc393 == '' ]]
+ return 0
+ plugn trigger service-action pre-create api
+ mkdir -p /var/lib/dokku/services/elasticsearch/api
+ mkdir -p /var/lib/dokku/services/elasticsearch/api/data
+ mkdir -p /var/lib/dokku/services/elasticsearch/api/config
+ touch /var/lib/dokku/services/elasticsearch/api/LINKS
+ service_commit_config api
+ declare SERVICE=api
+ local SERVICE_ROOT=/var/lib/dokku/services/elasticsearch/api
+ local CONFIG_VARIABLE=ELASTICSEARCH_CONFIG_OPTIONS
+ local ENV_VARIABLE=ELASTICSEARCH_CUSTOM_ENV
+ custom_env=
+ [[ -n '' ]]
+ [[ -n '' ]]
+ echo ''
+ config_options=
+ [[ -n '' ]]
+ [[ -n '' ]]
+ echo ''
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n elasticsearch ]]
+ echo elasticsearch
+ [[ -n 7.14.1 ]]
+ echo 7.14.1
+ write_database_name api
+ declare 'desc=write a sanitized database name'
+ declare SERVICE=api
+ local SERVICE_ROOT=/var/lib/dokku/services/elasticsearch/api
+ tr .- _
+ echo api
+ plugn trigger service-action post-create api
+ service_create_container api
+ local SERVICE=api
+ local SERVICE_ROOT=/var/lib/dokku/services/elasticsearch/api
+ local SERVICE_HOST_ROOT=/var/lib/dokku/services/elasticsearch/api
++ get_service_name api
++ declare 'desc=retrieve a docker service label'
++ declare SERVICE=api
++ echo dokku.elasticsearch.api
+ local SERVICE_NAME=dokku.elasticsearch.api
++ echo 7.14.1
++ cut -d . -f 1
+ local ELASTICSEARCH_MAJOR_VERSION=7
+ [[ -f /var/lib/dokku/services/elasticsearch/api/CONFIG_OPTIONS ]]
++ cat /var/lib/dokku/services/elasticsearch/api/CONFIG_OPTIONS
+ export CONFIG_OPTIONS=
+ CONFIG_OPTIONS=
+ [[ -f /var/lib/dokku/services/elasticsearch/api/SERVICE_MEMORY ]]
+ [[ -n '' ]]
+ [[ -f /var/lib/dokku/services/elasticsearch/api/SHM_SIZE ]]
+ [[ -n '' ]]
+ [[ -f /var/lib/dokku/services/elasticsearch/api/IMAGE ]]
++ cat /var/lib/dokku/services/elasticsearch/api/IMAGE
+ PLUGIN_IMAGE=elasticsearch
+ [[ -f /var/lib/dokku/services/elasticsearch/api/IMAGE_VERSION ]]
++ cat /var/lib/dokku/services/elasticsearch/api/IMAGE_VERSION
+ PLUGIN_IMAGE_VERSION=7.14.1
+ [[ 7 -eq 2 ]]
+ dokku_log_info2 'Extracting config files'
+ declare 'desc=log info2 formatter'
+ echo '=====> Extracting config files'
=====> Extracting config files
++ docker create elasticsearch:7.14.1
+ TEMP_DOCKER_CONTAINER_ID=b3ca59ad2a9bbdbf78a4445caf6c0bb2685285773cba0fde34c178fda293126b
+ docker cp b3ca59ad2a9bbdbf78a4445caf6c0bb2685285773cba0fde34c178fda293126b:/usr/share/elasticsearch/config/ -
+ tar -x -C /var/lib/dokku/services/elasticsearch/api
stat -: permission denied
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
```

**Comment by g1eyd50n** [comment]
same issue here with the same stack trace ^. I created a digital ocean droplet with dokku

**Comment by dozieogbo** [comment]
At this point, I am considering setting up a plain Docker ElasticSearch container.😔

**Comment by dozieogbo** [comment]
> same issue here @josegonzalez @dozieogbo with the same stack trace ^. I created a digital ocean droplet with dokku

I also created a DO droplet with Dokku

**Comment by g1eyd50n** [comment]
Thanks, @josegonzalez this fixed the issue.

**Comment by josegonzalez** [comment]
Great, I released this in 1.15.2.
